src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,421 @@
+            } else {
+                FileWriter f = new FileWriter(ikarosTaskData.getAbsoluteFile());
+                f.close();
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found. pls try reload again, or start a new");
+        } catch (IOException e) {
+            System.out.println("File corrupted. pls try reload again, or start a new");
+        } finally {
+            printList();
+            lineBreak();
+        }
+    }
+
+    /**
+     * Writes down the new tasks to the saved file
+     */
+    private static void SaveTasks() {
+        try {
+            FileWriter fw = new FileWriter("ikarosTaskData.txt");
+            for (Task task : taskList) {
+                fw.write(task.getTaskType() + ">"
+                        + task.getStatusIcon() + ">"
+                        + task.getDescription() + ">"
+                        + task.getDate() + System.lineSeparator());
+            }
+            fw.close();
+        } catch (IOException e) {
+            System.out.println("Something went wrong" + e.getMessage());
+        }
+    }
+
+    /**
+     * Checks and adds the tasks, from the saved .txt file
+     *
+     * @param command a String[] containing task information stored in the file
+     */
+    private static void loadingManager(String[] command) {
+        switch (command[0]) {
+        case "todo":
+            ToDo taskTodo = new ToDo(command[2]);
+            taskList.add(taskTodo);
+            break;
+        case "deadline":
+            Deadlines taskDeadline = new Deadlines(command[2], command[3]);
+            taskList.add(taskDeadline);
+            break;
+        case "event":
+            Event taskEvent = new Event(command[2], command[3]);
+            taskList.add(taskEvent);
+            break;
+        default:
+            System.out.println("file error");
+        }
+        if (command[1].equalsIgnoreCase("X")) {
+            taskList.get(taskList.size() - 1).markAsDone();
+        }
+    }
+
+    /**
+     * Checks to see which command the user has input
+     *
+     * @param command  the set of instructions the user input in a String array
+     * @param response the string of the user input
+     */
+    private static void instructionManager(String[] command, String response) {
+        try {
+            if (command[0].equalsIgnoreCase("add")) {
+                taskManager(command, response);
+            } else {
+                remove(command);
+            }
+        } catch (RemoveException e) {
+            System.out.println(e.getMessage());
+        } finally {
+            lineBreak();
+        }
+    }
+
+    /**
+     * Deciphers the specific task that the user wants to add to the list
+     *
+     * @param command  the set of instructions the user input in a String array
+     * @param response the string of the user input
+     */
+    private static void taskManager(String[] command, String response) {
```

Comment:
method names should ideally start with a verb. `taskManager` sounds like a noun. Methods above have the same issue